ui/utils.py

```diff
@@ -457,3 +460,36 @@ def get_error_response_summary_dict(response):
     else:
         summary_dict = {"content": response.text}
     return {"code": response.status_code, "url": response.url, **summary_dict}
+
+
+def now_in_utc():
+    """
+    Get the current time in UTC
+
+    Returns:
+        datetime.datetime: A datetime object for the current time
+    """
+    return datetime.datetime.now(tz=pytz.UTC)
+
+
+def send_refresh_request(full_api_url):
```

Comment:
This could use some testing coverage.  Something like this might work:

```python
def test_send_refresh_request(mocker, settings):
    client_secret = "secrets"
    client_id = "clientid"
    url = "http://test.url"
    settings.OPENEDX_API_CLIENT_ID = client_id
    settings.OPENEDX_API_CLIENT_SECRET = client_secret
    mock_post = mocker.patch("ui.utils.requests.post")
    send_refresh_request(url)
    expected_token_url = "{}/oauth2/access_token/".format(url)
    expected_data = {
        "grant_type": "client_credentials",
        "client_id": client_id,
        "client_secret": client_secret,
        "token_type": "JWT",
    }
    mock_post.assert_called_once_with(expected_token_url, data=expected_data)
```